2018-09-10-fotos-xcamp-2018.md

```diff
@@ -0,0 +1,10 @@
+---
+title:     Tolle Eindrücke und spannenden Teilnehmer beim XCamp 2018
+author:    Jens Bothmer
+published: 10.09.2018
+image:     
```

Comment:
We want the preview picture here too:

https://xcamp.autentity.net/wp-content/uploads/sites/18/2019/01/XCamp18-von.garnier.de-05636_resize.jpg